### PR TITLE
DFC 418 | Add language toggle

### PR DESCRIPTION
**Pull request by SophiaDWokoma**
### What changed

- Add a new feature flag in the env file called language_toggle
- Include a new css file into application.css
- Call 2 new functions from common express.
- Fix bug in build-sass command
- Document variable in README
- Add languageToggle key to translation files

### Why did it change

Add a new shared component called [one login language toggle](https://www.npmjs.com/package/@govuk-one-login/one-login-language-toggle)

### Issue tracking

- [DFC-418](https://govukverify.atlassian.net/browse/DFC-418)

### Environment variables or secrets

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

## Screenshots of view / Dev Testing
**Desktop**
Screenshot 2024-03-19 at 16 10 01
**Mobile**
Screenshot 2024-03-19 at 16 10 38

 ## Important 
 
 All pages are being translated apart from the title and hint text on this page  
Screenshot 2024-03-19 at 16 11 12

[DFC-419]: https://govukverify.atlassian.net/browse/DFC-419?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DFC-418]: https://govukverify.atlassian.net/browse/DFC-418?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/lib/config.js [hunk 1]
```diff
@@ -22,6 +22,7 @@ module.exports = {
       GA4_ENABLED: process.env.GA4_ENABLED || "false",
       GA4_CONTAINER_ID: process.env.GA4_CONTAINER_ID,
     },
+    LANGUAGE_TOGGLE_ENABLED: process.env.LANGUAGE_TOGGLE_ENABLED || false,
```
Comment (di-aholme) [inline src/lib/config.js:25]: Is false the correct default value here if we're using `0` and `1` to enable?

**Review by di-aholme — CHANGES_REQUESTED** [review]

File: src/lib/config.js (hunk 1 above)
Comment (SophiaDWokoma) [reply src/lib/config.js:25]: yeah I think you're right, it should be 0

Comment (SophiaDWokoma) [reply src/lib/config.js]: updated

File: src/assets/scss/application.scss [hunk 2]
```diff
@@ -2,3 +2,9 @@ $govuk-assets-path: "/public/";
 
 @import "../../../node_modules/govuk-frontend/govuk/all";
 @import "../../../node_modules/hmpo-components/all";
+@import "../../../node_modules/@govuk-one-login/one-login-language-toggle/stylesheet/styles";
+
+// style for language toggle
+.last-item {
```
Comment (Joanneyoung01) [inline src/assets/scss/application.scss:8]: This can be removed as the latest versionof the toggle handles the styling

Comment (SophiaDWokoma) [reply src/assets/scss/application.scss]: ahh ok I have removed now

**Review by Joanneyoung01 — DISMISSED** [review]

**Review by di-fabs — DISMISSED** [review]

**Review by merlinc — CHANGES_REQUESTED** [review]
This should be rebased over main, and a more recent version of `cri-common-express` used.

**Comment by SophiaDWokoma** [comment]
> This should be rebased over main, and a more recent version of `cri-common-express` used.

Hi @merlinc , thank you for your comments so far but this PR is still in draft. When it is ready to review it will be opened and I will let the you know via the collaboration channel.

**Review by liamStockwellGDS — APPROVED** [review]
looks good to me

**Review by di-aholme — APPROVED** [review]

**Pull request merged**